infra/makemake/default.nix

```diff
@@ -0,0 +1,13 @@
+{inputs}:
```

Comment:
I don't get it. This file doesn't seem to be used anywhere? `flake.nix` refers to `infra/makemake/configuration.nix` directly, see: https://github.com/ngi-nix/ngipkgs/pull/265/files#diff-206b9ce276ab5971a2489d75eb1b12999d4bf3843b7988cbe8d687cfde61dea0R155

Reply:
Maybe I missed something, have to take another look at the whole thing.